Packs/Tripwire/Integrations/Tripwire/Tripwire.yml

```diff
@@ -0,0 +1,538 @@
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Tripwire is a file integrity management (FIM), FIM monitors files and
+  folders on systems and is triggered when they have changed.
+display: Tripwire
+name: Tripwire
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: Versions IDs given comma seperated.
+      isArray: true
+      name: version_oids
+      required: false
+      secret: false
+    - default: false
+      description: Elements IDs of elements versions to fetch, comma seperated.
+      isArray: true
+      name: element_oids
+      required: false
+      secret: false
+    - default: false
+      description: Element names of elements versions to fetch. (case insensitive)
+        .comma seperated
+      isArray: true
+      name: element_names
+      required: false
+      secret: false
+    - default: false
+      description: Nodes IDs of elements versions to fetch. comma seperated.
+      isArray: true
+      name: node_oids
+      required: false
+      secret: false
+    - default: false
+      description: Nodes names of elements versions to fetch. comma seperated.
+      isArray: true
+      name: node_names
+      required: false
+      secret: false
+    - default: false
+      description: Rules IDs of elements versions to fetch. comma seperated.
+      isArray: true
+      name: rule_oids
+      required: false
+      secret: false
+    - default: false
+      description: Rules names of elements versions to fetch. comma seperated.
+      isArray: true
+      name: rule_names
+      required: false
+      secret: false
+    - default: false
+      description: Possible Hashes value (md5, sha1, sha256, sha512) of elements versions
+        to fetch. comma seperated.
+      isArray: true
+      name: version_hashes
+      required: false
+      secret: false
+    - default: false
+      description: Last baseline versions of elements versions to fetch. comma seperated.
+      isArray: true
+      name: baseline_version_ids
+      required: false
+      secret: false
+    - default: false
+      description: "Time detected range of element versions to fetch.  comma seperated.\
+        \ \nThe first argument should be the start of the range and the second is\
+        \ the range.\nThe format of the start and end can be either relative e.g.\
+        \ \"2 days, 1 day\" or date time \"2020-11-24T17:07:27Z,2020-11-25T17:07:27Z\""
+      isArray: true
+      name: time_detetcted_range
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Time received range of element versions to fetch. comma seperated.
+        The first argument should be the start of the range and the second is the range.
+        The format of the start and end can be either relative e.g. "2 days, 1 day" or date time "2020-11-24T17:07:27Z,2020-11-25T17:07:27Z"
+      isArray: true
+      name: time_received_range
```

Comment:
i'm contemplating if it is better to separate it to 2 args. what happens if the give you 3 or more args? it is a bit crunchy.